<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeoMapViewerForm</class>
 <widget class="QDialog" name="GeoMapViewerForm">
  <property name="windowTitle">
   <string>Карта и геоданные</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <widget class="QLabel" name="labelNote">
     <property name="text"><string>Выберите слой и зону для просмотра:</string></property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="layerSelector"/>
   </item>
   <item>
    <widget class="QPushButton" name="buttonViewMap">
     <property name="text"><string>Показать на карте</string></property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="dataTable"/>
   </item>
   <item>
    <widget class="QPushButton" name="buttonClose">
     <property name="text"><string>Закрыть</string></property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>